src/cljs/athens/electron.cljs

```diff
@@ -315,6 +317,42 @@
             mtime (or mtime1 (.. fs (statSync filepath) -mtime))]
         (assoc db :db/mtime mtime))))
 
+  (reg-event-fx
+    :local-storage/set-rfdb
+    [(inject-cofx :local-storage-map {:ls-key "theme/dark"
+                                      :key    :theme-dark?})
+     (inject-cofx :local-storage-map {:ls-key "appearance/width"
+                                      :key    :appearance-width})]
```

Comment:
Overall, using cofx with localStorage here isn't a bad idea, but having two keys (one for local-storage and which more typical edn keyword) doesn't seem like a good idea. I know you didn't write `:local-storage-map`, but I wonder if there's a better way to have the same key for both localStorage and clojure.

Reply:
I Will update after looking into this.